apps/src/feedback.js

```diff
@@ -1327,7 +1328,7 @@ FeedbackUtils.prototype.showSimpleDialog = function(options) {
       <DialogButtons
         confirmText={options.confirmText}
         cancelText={options.cancelText}
-        cancelButtonClass={options.cancelButtonClass}
+        isDangerCancel={!!options.isDangerCancel}
```

Comment:
Can you explain why this is was called a "cancel" button? Usually, I would expect that a cancel button goes back without making any changes. Isn't that the "Keep" button in this case? Or does "cancel" just  mean it's the secondary/non-default button?

Reply:
well, that's a good question :D 
The main problem here is that we already have a set of dialogs and button defined in blockly. While doing Phase 1.5 I was simply updating the cdo styles expecting all the dialogues to be consistent. But it turned out that they are not as consistent and not always used correct naming/structure/styles, just like the button I'm working on in this pr. It was a cancel button that was painted in `red` with the help of injected css with `!important`.

So, in short, the idea here is to fix and simplify where possible what we have right now without going to deep and rewriting too much of cdo-blockly (keeping in mind that we're going to get rid of it at some place)